src/couch/src/couch_auth_cache.erl

```diff
@@ -203,7 +203,8 @@ handle_call({fetch, UserName}, _From, State) ->
     [] ->
         couch_stats:increment_counter([couchdb, auth_cache_misses]),
         Creds = get_user_props_from_db(UserName),
-        State1 = add_cache_entry(UserName, Creds, erlang:now(), State),
+        ATime = couch_time_compat:unique_integer([monotonic, positive]),
```

Comment:
I put it here  mainly to hide the ifdef ugliness. But I can move it to couch_util

Though we should probably rename it as well to unique_monotonic_integer since  unique_integer/0 is not guaranteed to be always increasing, in theory could jump around like -10, then 100, then -5 etc.